src/_xoConnection.js

```diff
@@ -127,6 +127,16 @@ class XoConnection extends Xo {
     for (let n = disposers.length - 1; n > 0; ) {
       const params = disposers[n--];
       const method = disposers[n--];
+      if (method === "vm.delete") {
+        for (const id in this.objects.all) {
+          if (this.objects.all[id].other) {
+            const { "xo:backup:vm": backupVm } = this.objects.all[id].other;
```

Comment:
We don't want to delete backups not created by `xo-server-test`.

Reply:
No, after that, we test if the vm to delete is the right one. (`backupVm === params.id`)

Reply:
What if we create manually a backup for this VM?